doc/visual_testing/reporter_visual_inspection_suite.py

```diff
@@ -1,57 +1,63 @@
-"""Contain a bunch of functions run via __main__().
```

Comment:
Where can we see the result of this build ?

Reply:
Should be viewable if we ask for a full build of the doc in the PR otherwise it will be build only after a full build in main?

https://nilearn.github.io/stable/development.html#building-documentation

```
git commit --allow-empty -m "[full doc] request full build"
```

Will trigger a full build on this PR.

Do you think that this should be treated like the examples, that is we trigger a build of only the examples that have been touched by a PR?

Reply:
@bthirion 

to view the GLM reports as part of the doc

https://output.circle-artifacts.com/output/job/b7e150cb-ce03-42b8-b6f3-a1105c7a66d4/artifacts/0/dev/modules/generated_reports/glm_reports_examples.html

Reply:
> Do you think that this should be treated like the examples, that is we trigger a build of only the examples that have been touched by a PR?

Makes sense to me.